### PR TITLE
First draft of designer post

### PR DESCRIPTION
**Pull request by AaronO**

### CODE REVIEW

File: _posts/2016-19-05-designer.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title:  "UI/UX Designer"
+date:   2016-19-05 17:32:00
+categories: design
+type: Regular (Full Time)
+location: Lyon, France
+---
+
+As a designer you'll lead our UI/UX work. Helping the team deliver a kickass product. Our ideal candidate is passionate about product and building a beautiful yet easy to use experience for all our users. As our first non-technical hire you can have a massive impact. It's an exciting but challenging opportunity to be part of a growing startup with a passionate founding team.
```
Comment (Soreine) [inline _posts/2016-19-05-designer.md]: Rewriting "As our first non-technical hire you can have a massive impact" to "As the first designer to join us, you can have a massive impact" makes less assumption about the candidate (he could have technical skills too, which could be desired etc.).

File: _posts/2016-19-05-designer.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+### Responsibilities:
+* Design and maintain coherent design UX across all our products (website, web editor, desktop editor)
```
Comment (Soreine) [inline _posts/2016-19-05-designer.md]: "Design and maintain coherent design UX"
Either I don't understand the sentence, or there is a problem in the structure. Do you mean "Design and maintain a coherent UX ..." ?

File: _posts/2016-19-05-designer.md (hunk 1 above)
Comment (AaronO) [inline _posts/2016-19-05-designer.md]: Yep good point, I added "Git, GitHub or GitBook knowledge" under "Bonus Skills". Je vais changer `non-technical` vers `non-engineering`

File: _posts/2016-19-05-designer.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+* Design & prototype new features
+
+### Skills/Experience Required:
```
Comment (Soreine) [inline _posts/2016-19-05-designer.md]: -> Relevant Skills/Experience
This sounds more open minded. The tools are not requirements, only the ability to work with HTML/CSS and create mockups is.

File: _posts/2016-19-05-designer.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+* Photoshop/Sketch
+* HTML/CSS
+* Mockups
+
```
Comment (Soreine) [inline _posts/2016-19-05-designer.md]: + Communication skills (with the team and with users)

File: _posts/2016-19-05-designer.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+### Bonus skills:
+* JavaScript experience
+* Knowledge of modern frontend tools (React, Less, Browserify, ...)
+
+### Perks and Benefices:
```
Comment (Soreine) [inline _posts/2016-19-05-designer.md]: "Benefits", I guess ?

**Pull request merged**